dojo/tools/nessus/parser.py

```diff
@@ -197,6 +197,10 @@ def get_findings(self, file, test):
                     cwe = None
                     if item.findtext("cwe"):
                         cwe = item.find("cwe").text
+                    cvssv3 = None
+                    if item.findtext("cvss3_vector"):
+                        cvssv3 = CVSS3(str(item.find("cvss3_vector").text)).clean_vector(output_prefix=False)
```

Comment:
the attribute `cvssv3` need the prefix.
Also please use the form `item.findtext("cvss3_vector")` directly

Reply:
> Also please use the form item.findtext("cvss3_vector") directly
What do you mean by that?

Reply:
Something like this:
```Python
                    cvssv3 = None
                    if item.findtext("cvss3_vector"):
                        cvssv3 = CVSS3(item.findtext("cvss3_vector")).clean_vector()
```

Reply:
@damiencarol Thank you. I made those changes. 

However that does mean that the csv parser in the same file is wrong because it doesn't return the prefix, which is why I was confused when you mentioned it should.